File: core/src/main/java/in/koreatech/koin/core/networks/responses/UserInfoEditResponse.java
package in.koreatech.koin.core.networks.responses;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class UserInfoEditResponse {

    @SerializedName("success")
    @Expose
    public String success;

    @SerializedName("error")
    @Expose
    public String error;


    @SerializedName("message")
    @Expose
    public String message;

}

File: core/data/src/main/java/in/koreatech/core/networking/response/user/UserInfoEditResponse.kt
package `in`.koreatech.core.networking.response.user

import com.google.gson.annotations.SerializedName

data class UserInfoEditResponse(
    @SerializedName("success") val success: String,
    @SerializedName("error") val error: String,
    @SerializedName("message") val message: String
)
